### ISSUE TITLE
Handling String Scalars

### ISSUE THREAD

**Issue by emtwo** [mozilla/glam#565]
Right now there are some scalars that are strings, e.g. `a11y_instantiators`. These are not being processed by the ETL as of now and from what I can tell, they don't exist in tmo/MDV1 either.

Instead of showing "Glam hit an unknown bug" for these, perhaps we need to remove the probes from the search list for now.

**Comment by rafrombrc** [comment]
We already have a "this probe type is not supported" page for boolean scalars (see [a11y_sitezoom](https://glam.telemetry.mozilla.org/firefox/probe/a11y_sitezoom/explore?process=parent), we should double check that this page shows up for string scalars. Unfortunately, a11y_instantiators no longer seems to exist, so we need to find some other string scalars to test with.

**Comment by alekhyamoz** [comment]
deletion.request.context_id - GLAM dev shows as inactive. 
However, the probe dictionary shows this probe as 'never expires'
https://probes.telemetry.mozilla.org/?view=detail&probeId=scalar%2Fdeletion.request.context_id

**Comment by Iinh** [comment]
> Unfortunately, a11y_instantiators no longer seems to exist, so we need to find some other string scalars to test with.

Some example string scalars that I found:
https://probes.telemetry.mozilla.org/?view=detail&probeId=scalar%2Fblocklist.mlbf_generation_time
https://probes.telemetry.mozilla.org/?view=detail&probeId=scalar%2Fdeletion.request.context_id
https://probes.telemetry.mozilla.org/?search=a11&view=detail&probeId=scalar%2Fa11y.instantiators

However, for some reason these probes are not available on prod. But on dev, because now that we're using the Glean Dictionary search API, we can search for them. 

> deletion.request.context_id - GLAM dev shows as inactive.
> However, the probe dictionary shows this probe as 'never expires'
> https://probes.telemetry.mozilla.org/?view=detail&probeId=scalar%2Fdeletion.request.context_id

This looks like a bug to me. The current implementation of legacy telemetry search in the Glean Dictionary relies on the [public API of all the probes are recorded in the process](https://github.com/mozilla/glean-dictionary/blob/62b0a1b0e8b020c0c3230c011219c02377f68309/etl/firefox_legacy_etl.py?_pjax=%23js-repo-pjax-container%3Afirst-of-type%2C%20div%5Bitemtype%3D%22http%3A%2F%2Fschema.org%2FSoftwareSourceCode%22%5D%20main%3Afirst-of-type%2C%20%5Bdata-pjax-container%5D%3Afirst-of-type#L11-L14) as the source of truth for whether or not a probe is active. `deletion.request.context_id` is not listed on this page, which renders its status as inactive in the search API. See: [code](https://github.com/mozilla/glean-dictionary/blob/62b0a1b0e8b020c0c3230c011219c02377f68309/etl/firefox_legacy_etl.py?_pjax=%23js-repo-pjax-container%3Afirst-of-type%2C%20div%5Bitemtype%3D%22http%3A%2F%2Fschema.org%2FSoftwareSourceCode%22%5D%20main%3Afirst-of-type%2C%20%5Bdata-pjax-container%5D%3Afirst-of-type#L54-L58) and [search API result](https://dictionary.telemetry.mozilla.org/.netlify/functions/metrics_search_firefox_legacy?search=deletion_request).

Next action steps would be:
- Fix the incorrect status bug in the Glean Dictionary search API (further discussion on how to fix this is needed, but it's more of a Glean Dictionary issue so I'll open a bug for this issue in the that repo)
- Once the above is done, we can then display a custom error message for string scalars, specifying that we don't currently aggregate these probes.

As it seems we would need more time than what the M4 deadline allows to complete the work required, I propose we move this issue to M5.

**Comment by Iinh** [comment]
String scalars are now available on GLAM. For example:

- https://glam.telemetry.mozilla.org/firefox/probe/blocklist_mlbf_generation_time/explore?currentPage=1
- https://glam.telemetry.mozilla.org/firefox/probe/deletion_request_context_id/explore?currentPage=1
- https://glam.telemetry.mozilla.org/firefox/probe/a11y_instantiators/explore?currentPage=1

Most of them are marked as inactive. After investigation, I can confirm the "inactive" status is not a Glean Dictionary bug like initially suspected. Checking on STMO for this probe for the last 7 days, I only see NULL values being sent.

Since the "inactive" status is a separate issue, and as string scalars now are available on GLAM, I'm closing this ticket.
